pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[build-system]
+requires = [
+    "setuptools >= 44.1.1",
+    "setuptools_scm[toml] >= 4, <6",
+    "wheel >= 0.36",
+]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
+write_to = "blur_python_example/version.txt"
```

Comment:
Add `version_scheme = "release-branch-semver"` to this section to force the expected version increment